src/main/java/de/tum/in/www1/artemis/web/rest/admin/AdminBuildJobQueueResource.java

```diff
@@ -77,11 +77,26 @@ public ResponseEntity<List<LocalCIBuildJobQueueItem>> getRunningBuildJobs() {
      */
     @GetMapping("build-agents")
     @EnforceAdmin
-    public ResponseEntity<List<LocalCIBuildAgentInformation>> getBuildAgentInformation() {
+    public ResponseEntity<List<LocalCIBuildAgentInformation>> getBuildAgentSummary() {
         log.debug("REST request to get information on available build agents");
-        List<LocalCIBuildAgentInformation> buildAgentInfo = localCIBuildJobQueueService.getBuildAgentInformation();
-        // TODO: convert into a proper DTO and strip unnecessary information, e.g. build config, because it's not shown in the client and contains too much information
-        return ResponseEntity.ok(buildAgentInfo);
+        // remove the recentBuildJobs from the response
```

Comment:
Do we need this comment? This is more something for the JavaDoc of the called method.